src/main/java/com/calculator/app/Calculator.java

```diff
@@ -0,0 +1,23 @@
+package com.calculator.app;
+
+public class Calculator {
+
+    public int addition(int a, int b) {
+        return a + b;
+    }
+
+    public int subtraction(int a, int b) {
+        return a - b;
+    }
+
+    public int multiplication(int a, int b) {
+        return a * b;
+    }
+
+    public double division(int a, int b) {
+        if (b == 0) {
+            throw new ArithmeticException("Division by zero");
+        }
+        return 1.0 * a / b;
```

Comment:
```
        if (b == 0) {
            throw new ArithmeticException("Division by zero");
        }

        return 1.0 * a / b;
```
Try to structure your code in methods.

Like you divide functionality to 5 small methods, not 1 large.
In method you also divide code into logic blocks.

       if (b == 0) {
            throw new ArithmeticException("Division by zero");
        }
        That's one logic block, we validating input and throw exception
        
        return 1.0 ...
         it's other block, we perform divide and return.
         
          So let's just show it, by separating code blocks by empty line. 
           It's more easier to read, if I want to read about return, why I should read about exception validation and find when this block is ends.
           
            If I could see it immidietly.
            
             1 class is like book page.
              Just by empty line mark different paragraph, it's easier to read and focus on what you need.